sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,8 +1,6 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
-
-### Features Added
+## 1.0.0-beta.5 (2024-11-18)

```

Comment:
I hope we can get CI to green tomorrow, and this is a reminder just in case for us to not forget to update the eventual release date.
```suggestion
## 1.0.0-beta.5 (2024-11-19)

```

Reply:
Thank you for keeping this up to date :)